app/build.gradle

```diff
@@ -266,8 +269,39 @@ dependencies {
     testImplementation "androidx.room:room-testing:2.2.6"
     testImplementation "org.jetbrains.kotlinx:kotlinx-coroutines-test:1.3.9"
     testImplementation "androidx.arch.core:core-testing:2.1.0"
-    androidTestImplementation "com.android.support.test:runner:1.0.2"
-    androidTestImplementation "com.android.support.test.espresso:espresso-core:3.0.2"
+    androidTestImplementation "io.insert-koin:koin-test-junit4:3.0.2"
+    testImplementation 'org.hamcrest:hamcrest-all:1.3'
+
+
+    // Core library
+        //androidTestImplementation("androidx.test:core-ktx:1.5.0")
+
+        // AndroidJUnitRunner and JUnit Rules
+        androidTestImplementation("androidx.test:runner:1.5.1")
+        androidTestImplementation("androidx.test:rules:1.5.0")
+
+        // Assertions
+        androidTestImplementation("androidx.test.ext:junit-ktx:1.1.4")
+        androidTestImplementation("androidx.test.ext:truth:1.5.0")
+
+        // Espresso dependencies
+        androidTestImplementation( "androidx.test.espresso:espresso-core:3.5.0")
+        androidTestImplementation( "androidx.test.espresso:espresso-contrib:3.5.0")
+        androidTestImplementation( "androidx.test.espresso:espresso-intents:3.5.0")
+        androidTestImplementation( "androidx.test.espresso:espresso-accessibility:3.5.0")
+        androidTestImplementation( "androidx.test.espresso:espresso-web:3.5.0")
+        androidTestImplementation( "androidx.test.espresso.idling:idling-concurrent:3.5.0")
+
+        // The following Espresso dependency can be either "implementation",
+        // or "androidTestImplementation", depending on whether you want the
+        // dependency to appear on your APK"s compile classpath or the test APK
+        // classpath.
+        androidTestImplementation( "androidx.test.espresso:espresso-idling-resource:3.5.0")
+
+        debugImplementation "androidx.fragment:fragment-testing:1.5.5"
+
+        debugImplementation "androidx.test:monitor:1.6.0"
```

Comment:
Please undo all of this. There is no need for it.

Reply:
Ok noted. I will update as stated